RabbitMQMessage/RabbitMQMessageQueue.cs

```diff
@@ -0,0 +1,43 @@
+﻿using RabbitMQ.Client;
+using System.Text;
+namespace ServerMonitoringSystem.RabbitMQMessage
+{
+    public class RabbitMQMessageQueue : IMessageQueue
+    {
+        private readonly string _hostName;
+        private readonly string _queueName;
+
+        public RabbitMQMessageQueue(string hostName, string queueName)
+        {
+            _hostName = hostName;
+            _queueName = queueName;
+        }
+
+        public void Publish(string message)
+        {
+            var factory = new ConnectionFactory()
+            {
+                HostName = _hostName,
+                UserName = "guest",
```

Comment:
same comment for the RabbitMQ connection configs, let's add them to the configurations